scripts/migration/migrate_menas.py

```diff
@@ -0,0 +1,433 @@
+#!/usr/bin/python3
+
+# Copyright 2018 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import secrets  # migration hash generation
+
+from minydra.dict import MinyDict  # dictionary with dot access
+from pymongo import MongoClient
+from pymongo.database import Database
+from pymongo.write_concern import WriteConcern
+from pymongo.read_concern import ReadConcern
+from pymongo.errors import DuplicateKeyError
+from typing import List
+
+# python package needed to run:
+# pymongo minydra
+# so install those e.g. by running
+# pip install pymongo minydra
+
+# Default configuration
+# =====================
+
+defaults = MinyDict({
+    'verbose': False,
+    'dryrun': False,
+    'target_db_name': "menas"
+})
+
+migration_hash = secrets.token_hex(3)  # e.g. 34d4e10f
+
+SOURCE_DB_NAME = "menas"
```

Comment:
feels a bit weird for this to be not part of the input

Reply:
My assumption here was that while the user might want to change the target_db (e.g. for a careful migration/tryout), the source db will always be `menas`. Is there a case where it may not be?

If so, it is relatively easy to add an option to specify it

Reply:
I have added the option to define the source db name, too, it is `-s`/`--source-database`. Defaults to `menas`.